src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -1077,12 +1077,24 @@ private Task<BatchAccount> CreateBatchAccountAsync()
 
         private Task<IResourceGroup> CreateResourceGroupAsync()
```

Comment:
1. Trim will fail if tags are null (see the other comment).
2. The code can be modified to avoid duplication:

	var tags = !string.IsNullOrWhiteSpace(configuration.Tags) ? DelimitedTextToDictionary(configuration.Tags, "=", ",") : null;

	var resourceGroupDefinition = azureSubscriptionClient
		.ResourceGroups
		.Define(configuration.ResourceGroupName)
		.WithRegion(configuration.RegionName);
		
	resourceGroupDefinition = tags != null ? resourceGroupDefinition.WithTags(tags) : resourceGroupDefinition;

	return Execute(
		$"Creating Resource Group: {configuration.ResourceGroupName}...",
		() => resourceGroupDefinition.CreateAsync());